tests/operator_test.go

```diff
@@ -1365,8 +1403,13 @@ spec:
 			startAllVMIs(migratableVMIs)
 
 			// Update KubeVirt from the previous release to the testing target release.
-			By("Updating KubeVirtObject With Current Tag")
-			patchKvVersionAndRegistry(kv.Name, curVersion, curRegistry)
+			if updateOperator {
+				By("Updating virt-operator installation")
+				installOperator(curOperatorManifestPath)
+			} else {
```

Comment:
This `else` looks suspicious. Shouldn't the part in the `else` clause always be executed?

Reply:
As far as I understand - no - since as long as the tag/registry aren't specified in the KubeVirt CR, the operator defaults them to its own image's tag/registry - which is what we want here. This is how it's described in the [user guide](https://kubevirt.io/user-guide/operations/updating_and_deletion/#updating-kubevirt-control-plane):
```
When no imageTag value is set in the kubevirt CR, the system assumes that the version of KubeVirt is locked to the version of the operator. This means that updating the operator will result in the underlying KubeVirt installation being updated as well.
```

Reply:
Yes. I see now that you made that conditional above. So looks sane.